packages/components/src/local/route/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactNode } from 'react'
+import { LayerGroup, Polyline } from 'react-leaflet'
+
+import { routesFor} from './helpers/routes-for'
```

Comment:
oh, one more tiny one 
```suggestion
import { routesFor } from './helpers/routes-for'
```